packages/test/src/test-integration-workflows.ts

```diff
@@ -1264,3 +1264,37 @@ export const interceptors: workflow.WorkflowInterceptorsFactory = () => {
   }
   return {};
 };
+
+export async function completableWorkflow(completes: boolean): Promise<void> {
+  await workflow.condition(() => completes);
+}
+
+test('Count workflow executions', async (t) => {
+  const { taskQueue, createWorker, executeWorkflow, startWorkflow } = helpers(t);
+  const worker = await createWorker();
+  const client = t.context.env.client;
+
+  await worker.runUntil(async () => {
```

Comment:
In this specific case, it would be easy to design this test to not even need to have an actual Workflow function and to start a Workflow Worker.

Instead of passing a Workflow function to `startWorkflow`, you may provide a raw string (when passing a workflow function, the SDK will internally take the name of that function). Then instead of (or in addition to?) grouping by `ExecutionStatus`, group by `WorkflowType`.

That will make the test slightly faster. Not a huge difference, but those small things eventually cumulate to something significant.

Reply:
Turns out we can only `GROUP BY` on `ExecutionStatus`, this is what I get when I try to `GROUP BY` on `WorkflowType`:
```
invalid query: operation is not supported: \'group by\' clause is only supported for ExecutionStatus search attribute'
```

Because of this, I'm going to keep the test case as is, its seems to be the only way to test that we get the groupings & group values we expect.

Reply:
I'd thought I had run this prior to pushing, does it not make code changes, just warnings?